client/web/src/enterprise/batches/create/RunServerSideModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+import { VisuallyHidden } from '@reach/visually-hidden'
+import CloseIcon from 'mdi-react/CloseIcon'
+
+import { Button, Link, Modal } from '@sourcegraph/wildcard'
+
+import styles from './RunServerSideModal.module.scss'
+
+export interface RunServerSideModalProps {
+    name: string
+    originalInput: string
+    isLightTheme: boolean
+    setIsRunServerSideModalOpen: (condition: boolean) => void
+}
+
+export const RunServerSideModal: React.FunctionComponent<RunServerSideModalProps> = ({
+    name,
+    originalInput,
+    isLightTheme,
+    setIsRunServerSideModalOpen,
+}) => (
+    <Modal
+        onDismiss={() => {
+            setIsRunServerSideModalOpen(false)
+        }}
+        aria-labelledby={MODAL_LABEL_ID}
+        className={styles.modal}
+    >
+        <div>
+            <h3 id={MODAL_LABEL_ID}>Running batch changes server side is not enabled</h3>
+            <Button
+                className={styles.close}
+                onClick={() => {
+                    setIsRunServerSideModalOpen(false)
+                }}
+            >
+                <VisuallyHidden>Close</VisuallyHidden>
+                <CloseIcon className={styles.icon} />
+            </Button>
+        </div>
+
+        <div className={styles.content}>
+            <div className={styles.left}>
+                <div className={styles.heading}>
+                    <p>
+                        Install executors to enable running batch changes server-side instead of locally. Executors can
+                        also be autoscaled to speed up creating large-scale batch changes.
+                    </p>
+                </div>
```

Comment:
I think we could just apply this style directly to the `<p />` here:

```suggestion
                <p className={styles.heading}>
                    Install executors to enable running batch changes server-side instead of locally. Executors can
                    also be autoscaled to speed up creating large-scale batch changes.
                </p>
```